docker/rootfs/etc/s6-overlay/s6-rc.d/go2rtc-healthcheck/run

```diff
@@ -0,0 +1,22 @@
+#!/command/with-contenv bash
+# shellcheck shell=bash
+# Start the go2rtc-healthcheck service
+
+set -o errexit -o nounset -o pipefail
+
+# Logs should be sent to stdout so that s6 can collect them
+
+# Give some additional time for go2rtc to start before start pinging
+sleep 10s
+echo "[INFO] Starting go2rtc healthcheck service..."
+
+while sleep 60s; do
+    # Check if the service is running
+    if ! timeout 10s curl -fsSL -o /dev/null http://127.0.0.1:1984/api/streams 2>&1; then
```

Comment:
Is there a good reason to not use the builtin timeout parameters for curl?

Reply:
No good reason, but there is probably no good reason not to use bash's timeout either.

Anyway, I'll change to curl.